python/ClusterLauncher/PBSJob.py

```diff
@@ -67,11 +67,13 @@ def copyFiles(self, job_file):
         # Copy directories
         if params.isValid('copy_files'):
             for file in params['copy_files'].split():
-                print file
                 if os.path.isfile('../' + file):
                     shutil.copy('../' + file, '.')
                 elif os.path.isdir('../' + file):
-                    shutil.copytree('../' + file, file)
+                    try:
+                        shutil.copytree('../' + file, file)
+                    except OSError:
+                        pass
```

Comment:
Because we are copying stuff from the same test directory into this new test directory multiple times... ignore when the directory already exists. Probably better to perform this as a logical check, rather than an error handler...